src/main/java/at/hannibal2/skyhanni/features/inventory/bazaar/BazaarCancelledBuyOrderClipboard.kt

```diff
@@ -16,16 +18,21 @@ import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
 class BazaarCancelledBuyOrderClipboard {
 
     private val patternGroup = RepoPattern.group("bazaar.cancelledorder")
+
+    /**
+     * REGEX-TEST: §6§7from §a59§7x §7missing items.
+     */
     private val lastAmountPattern by patternGroup.pattern(
         "lastamount",
-        "§a(?<amount>.*)§7x"
+        "§6§7from §a(?<amount>.*)§7x §7missing item" +
```

Comment:
Is there any particular reason on the + "" part

Reply:
No! Good catch.